website/content/docs/api-gateway/configuration/gateway.mdx

```diff
@@ -183,3 +183,49 @@ tls:
 
 ```
 
+#### Example cross-namespace certificateRef
```

Comment:
This section follows the pattern established by the corresponding cross-namespace docs for routes [here](https://www.consul.io/docs/api-gateway/configuration/routes#example-cross-namespace-backendref).

Reply:
Terrific. Thanks for adding this. I'm going to add a suggestion that, if we think is an improvement, we can apply to the routes docs, too.